recipes-graphics/imx-gpu-apitrace/imx-gpu-apitrace_7.1.0.bb

```diff
@@ -34,4 +34,4 @@ FILES_${PN} = "${bindir} ${libdir}"
 FILES_${PN}-dbg += "${libdir}/*/*/.debug"
 
 PACKAGE_ARCH = "${MACHINE_SOCARCH}"
-COMPATIBLE_MACHINE = "(mx6q|mx6dl|mx6sx|mx6sl|mx7ulp|mx8mq)"
+COMPATIBLE_MACHINE = "(mx6q|mx6dl|mx6sx|mx6sl|mx7ulp|mx8)"
```

Comment:
This should be another commit ... also, I'd prefer to not use generic overrides if possible ...